pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -296,7 +296,9 @@ stdenv.mkDerivation {
           echo ${libc_lib}/lib/32/ld-linux.so.2 > $out/nix-support/dynamic-linker-m32
         fi
 
-        local ldflagsBefore=(-dynamic-linker "$dynamicLinker")
+     if [ -n "''${dynamicLinker:-}" ]; then
```

Comment:
This second conditional is unneeded, no?

Reply:
It's already part of the first, unless I messed up the string interpolation.

Reply:
You're right, I missed the indentation. Pushing a revised patch now